index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <div class="header-tab">
+            <div class="tab-title">
+                Ammar Hamlaoui
+            </div>
+            <div class="menu-button">
+                <img src="src/union.png" alt="Menu icon"/>
+            </div>
+        </div>
+        <div class="header-content">
+            <div class="header-primary-text">
+                Hey There. <br> I'm Ammar
+            </div>
+            <div class="header-supporting-text">
+                I'm a Software Developer
+            </div>
+            <div class="header-description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="header-social-media">
```

Comment:
- [ ]  This is an unordered list, it is required to use the `<ul>` tag for the list and `<li>` for its elements, which should also redirect to your social links, meaning they should have an anchor tag (`<a>`).